src/internal/operators/onErrorResumeNext.ts

```diff
@@ -83,7 +83,11 @@ export function onErrorResumeNext<T, A extends readonly unknown[]>(
 export function onErrorResumeNext<T, A extends readonly unknown[]>(
   ...sources: [[...ObservableInputTuple<A>]] | [...ObservableInputTuple<A>]
 ): OperatorFunction<T, T | A[number]> {
-  const nextSources = argsOrArgArray(sources);
+  // For some reason, TS 4.1 RC gets the inference wrong here and infers the
+  // result to be `A[number][]` - completely dropping the ObservableInput part
+  // of the type. This makes no sense whatsoever. As a workaround, the type is
+  // asserted explicitly.
```

Comment:
Is it worth opening an issue with an MVP repro in the TS repo?

Reply:
If you can create a minimal reproduction, sure, go for it. I won't be; I've found interactions in the TS repo to be sufficiently discouraging that I no longer bother.

Reply:
You'll need to reproduce it in the nightly, BTW.

Reply:
> I've found interactions in the TS repo to be sufficiently discouraging that I no longer bother.

FWIW: I haven't had that experience. We've always had a really strong relationship with the TS team, and they've pushed a lot of really cool stuff through for us in the past.